### PR TITLE
Add Empty state component

### PR DESCRIPTION
**Pull request by camjc**
- Fix confusing variable name in Breadcrumb component
- Change module.export back to export default
  (Babel was buggy in the initial v6 release)

 
screen shot 2015-12-01 at 4 34 10 pm

### CODE REVIEW

File: test/components/alexandria/EmptyComponentTest.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* eslint-env node, mocha */
+/* global expect */
+/* eslint no-console: 0 */
+
+import createComponent from 'helpers/shallowRenderHelper';
+import EmptyComponent from 'components/alexandria/EmptyComponent.js';
```
Comment (antklim) [inline test/components/alexandria/EmptyComponentTest.js]: `.js` is not necessary

**Comment by antklim** [comment]
Awesome :100: 
LGTM

File: test/components/alexandria/EmptyComponentTest.js (hunk 1 above)
Comment (camjc) [inline test/components/alexandria/EmptyComponentTest.js]: Ah, I always forget about these. Cheers.

**Comment by etaoins** [comment]
:rocket:

**Pull request merged**